torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3794,6 +3813,22 @@ def sample_inputs_fill_(op_info, device, dtype, requires_grad, **kwargs):
     for shape, args in cases:
         yield SampleInput(make_arg(shape), args=args)
 
+def _fill_np(a, value):
+    return a.copy().fill(value)
+
+def _fill_aten(a, value):
+    t = a * False
+    with torch.no_grad():
+        t.fill_(value)
+    return t
+
+def _fill_sample_kwargs(device, dtype, input):
+    if dtype is torch.bool:
+        value = True
+    else:
+        value = 3
+
+    return ({'value': value}, {'value': value})
```

Comment:
why 2 tuple elements?

Reply:
This is super weird and we could sugar over this, but it's because we sometimes pass different arguments to the NumPy op, so we have "torch kwargs" and "NumPy kwargs" here